spec/classes/ntp_spec.rb

```diff
@@ -29,6 +29,12 @@
         it { should contain_file('/etc/ntp.conf').with_group('0') }
         it { should contain_file('/etc/ntp.conf').with_mode('0644') }
 
+        if system == 'RedHat' or system == 'Fedora'
```

Comment:
I've determined that this can also read `if :osfamily == 'RedHat'` to accomplish the same check.